fr/integrations/stream-connectors/sc-bsm.md

```diff
@@ -0,0 +1,126 @@
+---
+id: bsm
+title: BSM
+---
+
+## Ce qu'apporte l'intégration de BSM + Centreon 
+
+Le Stream Connector BSM envoie les données de **Centreon** vers le gestionnaire de service **Micro Focus BSM** (Business Service Management).
+
+## Comment ça marche
+
+* Chaque fois qu'un service ou un hôte est vérifié par le moteur de supervision, le résultat passe par *Centreon Broker*, qui utilise les fonctions définies dans le script du Stream Connector pour transmettre les changements d'état à BSM.
+
```

Comment:
Pas d'explication comme anglais :"* State changes can occur in case of an anomaly detection or metrics falling out of range." ?

Reply:
Je vais voir comment j'aligne ça.